cli/packages/cmd/export.go

```diff
@@ -111,6 +116,8 @@ func init() {
 	exportCmd.Flags().StringP("format", "f", "dotenv", "Set the format of the output file (dotenv, json, csv)")
 	exportCmd.Flags().Bool("secret-overriding", true, "Prioritizes personal secrets, if any, with the same name over shared secrets")
 	exportCmd.Flags().String("token", "", "Fetch secrets using the Infisical Token")
+	exportCmd.Flags().String("universal-auth-client-id", "", "Machine Identity universal auth client ID")
+	exportCmd.Flags().String("universal-auth-client-secret", "", "Machine Identity universal auth client secret")
 	exportCmd.Flags().StringP("tags", "t", "", "filter secrets by tag slugs")
```

Comment:
With this approach, users would be logging in each time but login operations are limited in most cases to reduce usage. We should separate using the token from token generation. For example if system A has access to the auth id and secret and system B needs to talk to infisical then system A can login and get a short lived token and pass it on to system B. This way, system B doesn't need to know the full credentials (least privilege).

`infisical login --method=universal-auth --client-id=<> --client-secret=<>`

default login method would be `user`. In the future we will have many more auth methods such as aws login, scim, etc. The token generated from this login can be passed in as `--token` like we already do.

I would also create a new sub command to show current token details and allow renew. For example:

`infisical token renew <token>` (renews the current token but throws error if it is service token type)
`infisical token summary` (show details like num used, time left to live)